projects/organization-management/src/app/components/group-form/group-form.component.ts

```diff
@@ -0,0 +1,39 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+import { HttpError } from 'ish-core/models/http-error/http-error.model';
+import { SelectOption } from 'ish-shared/forms/components/select/select.component';
+
+import { Node, NodeTree } from '../../models/node/node.model';
+
+@Component({
+  selector: 'ish-group-form',
+  templateUrl: './group-form.component.html',
+  changeDetection: ChangeDetectionStrategy.Default,
+})
+export class GroupFormComponent implements OnChanges {
+  @Input() form: FormGroup;
+  @Input() error: HttpError;
+  @Input() parents: NodeTree;
+  parentOptions: SelectOption[];
+
+  ngOnChanges(changes: SimpleChanges) {
+    if (changes.form || changes.parents) {
+      this.parentOptions = this.mapParentOptions(this.parents.nodes);
+    }
+  }
+
+  private mapParentOptions(nodeMap: { [id: string]: Node }): SelectOption[] {
+    return Object.keys(nodeMap)
+      .map(nodeId => ({ value: nodeId, label: nodeMap[nodeId].name }))
+      .sort((a: SelectOption, b: SelectOption) => {
+        if (a.label < b.label) {
+          return -1;
+        }
+        if (a.label > b.label) {
+          return 1;
+        }
+        return 0;
+      });
+  }
```

Comment:
The `keyvalue` pipe alone could not solve the mapping issue here so a solution with
```
      [options]="toOptions(parents.nodes | keyvalue)"
```
was introduced.
Unfortunately the additional function wrapper results in a select with options that does not show the selected option. This could not be resolved together with the `ish-select` component. So I reverted to an improved version without the `keyvalue` pipe for now.
Maybe the transition to Formly forms will provide a new reason to explore the `keyvalue` pipe options.